src/auth/socialAuth/facebook/css/Facebook.scss

```diff
@@ -0,0 +1,9 @@
+.kep-login-facebook{
+    padding: 5px;
+    border-radius: 0rem;
+}
+  
+.fa-facebook {
+    color: pink;
```

Comment:
I think you consider defining color in the `color.scss` file and reference it as a variable instead.